library/modules/bigip_gtm_wide_ip.py

```diff
@@ -209,6 +261,11 @@
   returned: changed
   type: list
   sample: ['alias1.foo.com', '*.wildcard.domain']
+persistence:
+  description: Whether the pool connections will be persisted.
+  returned: changed
+  type: str
+  sample: disabled
 '''
 
```

Comment:
where are the rest of returned values from returnables, document every value the module returns after it runs